### ISSUE TITLE
As a user I want to easily migrate my data from one pod to another without breaking outside links to my data

### ISSUE THREAD

**Issue by dsebastien** [solid/user-stories#9]
The last part (without breaking links) might be out of scope for the server, but it's probably worth discussing.

**Comment by dsebastien** [comment]
3xx redirections have been mentioned in previous discussions and of course that's a proved/working solution on the Web, but if we want the social aspect of Solid to work, I feel like link "stability" matters.

Moving your pod around should probably not break the links that people have created (unless you know what you're doing), and keeping an intermediary in the chain (i.e., the previous pod provider doing 3xx) might be a concern for privacy.

One idea that I've mentioned on Gitter, although somewhat crazy, might be to have a GTLD for Solid and an easy way to get your own names/identifiers there; those would then be your own stable entry points. I'm just thinking out loud here though.. :)

**Comment by dsebastien** [comment]
Interesting/related discussion and ideas: https://gitter.im/solid/chat?at=5c877a26c8e5bc51622a288e

**Comment by kjetilk** [comment]
Yeah, and #25 also stems from the same motivation. I know that myself, I wouldn't use a Pod under somebody else's domain name.

**Comment by Mitzi-Laszlo** [comment]
Added to readme

**Comment by megoth** [comment]
Opening this as part of [moving all user stories into issues again](https://github.com/solid/user-stories/issues/46#issuecomment-579773766).
